media/stylus/components/wiki/document-head.styl

```diff
@@ -3,6 +3,9 @@
     clear: both;
 
     h1 {
-        margin-bottom: 20px;
+        word-break: break-all;
+        line-height: 1.2;
+        margin-top: .1458em;
```

Comment:
Sorry, the vertical rhythm has been specified by the designer in pixels, we can't deviate from the design in one place without reconsidering all margins in the entire wiki and consulting with the design team. Please leave the original margins.